### PR TITLE
Task-47659: Projects not visible in space after editing space name

### PR DESCRIPTION
**Pull request by AzmiTouil**
To fix the problem we should get space by display name instead of get it by GroupId

### CODE REVIEW

**Review by ahamdi — DISMISSED** [review]
Could you add a UT for this change

File: integration/pom.xml [hunk 1]
```diff
@@ -24,7 +24,7 @@
   <parent>
     <artifactId>task-management-parent</artifactId>
     <groupId>org.exoplatform.addons.task</groupId>
-    <version>3.3.x-SNAPSHOT</version>
+    <version>3.3.x-maintenance-SNAPSHOT</version>
```
Comment (azayati) [inline integration/pom.xml]: this modifications should not appear IMHO, should be checked

**Review by azayati — CHANGES_REQUESTED** [review]
See review

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**